BUILD.md

```diff
@@ -25,7 +25,7 @@ Required Software Dependencies
 FreeOrion depends on the following software to build:
 
   * [Visual Studio] - 2022 for Windows Desktop ; Windows only
-  * [Xcode] - 10.1 or later ; MacOS only
+  * [Xcode] - 13.2 or later ; MacOS only
```

Comment:
https://github.com/freeorion/freeorion/actions/runs/4902766692/jobs/8754806597#step:7:28

says `Check for working CXX compiler: /Applications/Xcode_13.2.1.app`

Reply:
Looks like minimum available is `11.7` https://github.com/actions/runner-images/blob/main/images/macos/macos-11-Readme.md
Not sure how to enforce it.

Reply:
Looks like `xcode-select` should work: https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-ios/actions/runs/2741438390/workflow